Documentation/quickstart/debugging.rst

```diff
@@ -66,13 +66,13 @@ This will start a ``gdb`` server. Then, start ``gdb`` with:
 .. code-block:: console
 
   $ cd nuttx/
-  $ gdb-multiarch nuttx/nuttx
+  $ gdb-multiarch ./nuttx
 
 Inside ``gdb`` console, connect to the ``gdb`` server with:
 
 .. code-block::
 
-  (gdb) target extended-remote :3333
+  (gdb) tar remote :3333
```

Comment:
tar -> target

Reply:
@simbit18 I use "tar" here too, since GDB accept reduced from (tar: target; mon: monitor, etc). But maybe it is better to use the formal way

Reply:
Hi @acassis I also use the abbreviated form but I think for documentation the unabbreviated form is preferable.